managed/CounterStrikeSharp.API/Modules/Commands/Targeting/Target.cs

```diff
@@ -45,7 +46,7 @@ private static bool ConstTargetType(string target, out TargetType targetType)
     {
         targetType = TargetType.Invalid;
         slug = null!;
-        if (!target.StartsWith("#"))
+        if (!target.StartsWith('#'))
```

Comment:
?

Reply:
> ?

Isn't it better to use char instead of string with StartsWith?

Reply:
there is a slight difference in the performance but thats it